# Outgoing command to low level controller in encoder ticks / second
int16   linear_ticks_sec
int16   angular_ticks_sec
